<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addEntryDialog</class>
 <widget class="QDialog" name="addEntryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>81</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Entry</string>
  </property>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>80</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Entry</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="entryInput">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::VisualMoveStyle</enum>
   </property>
  </widget>
  <widget class="QLabel" name="entryLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>IP Address</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="wlButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>63</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Whitelist</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="blButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>63</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Blacklist</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>entryInput</tabstop>
  <tabstop>wlButton</tabstop>
  <tabstop>blButton</tabstop>
  <tabstop>addButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
